plugins/CustomPiwikJs/File.php

```diff
@@ -60,7 +60,7 @@ public function getName()
      */
     public function hasWriteAccess()
     {
-        return $this->hasReadAccess() && is_writable($this->file);
+        return is_writable($this->file);
```

Comment:
hopefully it's fine to just check whether file is writable. It solves the issue when the target file does not exist yet, hasWriteAccess should return true when  the file can be created/written to.